addons/noactions/stringtable.xml

```diff
@@ -0,0 +1,63 @@
+
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="TAC">
+    <Package name="No Actions">
+        <Key ID="STR_tac_noactions_settings">
```

Comment:
```suggestion
        <Key ID="STR_TAC_NoActions_Settings">
```
All strings should follow this type of naming style.

Additionally, settings category should be `STR_TAC_Armory_DisplayName -> No Actions` combined with `format ["TAC %1", localize LSTRING(DisplayName)]`.